### PR TITLE
Add NoisyTyperSwift 0.1

### PR DESCRIPTION
**Pull request by cades**
* Add NoisyTyper.app

* Add noisy-typer-swift.rb

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

**Comment by miccal** [comment]
Is this the same thing as your [other PR](https://github.com/caskroom/homebrew-cask/pull/28075)?

**Comment by cades** [comment]
@miccal, they are not the same. This one is an improved clone of the previous one. The previous one is classical and still working, but not maintained.

File: Casks/ialaddin-noisytyper.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'ialaddin-noisytyper' do
+  version '0.1'
+  sha256 '6f15783df37d7a75c6387c720e18fd188f82eb75dc84bac8768824c188ae5927'
+
+  url "https://github.com/iAladdin/NoisyTyper-Swift/archive/release/version-#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/ialaddin-noisytyper.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/ialaddin-noisytyper.rb (hunk 1 above)
Comment (cades) [inline Casks/ialaddin-noisytyper.rb]: OK, thanks!

File: Casks/ialaddin-noisytyper.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  appcast "https://github.com/iAladdin/NoisyTyper-Swift/releases.atom"
```
Comment (vitorgalvao) [inline Casks/ialaddin-noisytyper.rb]: This is incomplete. Please read the documentation I linked you to. It requires a `checkpoint`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Comment by miccal** [comment]
@vitorgalvao this is not distributed via Github releases, therefore the `appcast` is useless, no?

**Comment by vitorgalvao** [comment]
@miccal Yes, you’re correct. Misread the `url`.

**Pull request merged**